src/main/java/com/prgms/voucher/voucherproject/VoucherApp.java

```diff
@@ -0,0 +1,60 @@
+package com.prgms.voucher.voucherproject;
+
+import com.prgms.voucher.voucherproject.domain.VoucherType;
+import com.prgms.voucher.voucherproject.io.Console;
+import com.prgms.voucher.voucherproject.repository.MemoryVoucherRepository;
+import com.prgms.voucher.voucherproject.repository.VoucherRepository;
+import com.prgms.voucher.voucherproject.service.VoucherService;
+
+public class VoucherApp implements Runnable{
+    private final Console console = new Console();
+    private final VoucherService voucherService;
+    private static VoucherRepository voucherRepository;
+
+    public VoucherApp(VoucherService voucherService) {
+        this.voucherService = voucherService;
+        this.voucherRepository = voucherRepository;
+    }
+
+
+    public static void main(String[] args) {
+        voucherRepository = new MemoryVoucherRepository();
+        new VoucherApp(new VoucherService(voucherRepository)).run();
+    }
+
+    @Override
+    public void run() {
+        while(true){
+            console.printMenu();
+            switch (console.inputCommand().toLowerCase()){
+                case "exit":
+                    return;
+
+                case "create":
+                    console.printKindOfVoucher();
+                    int selectedNum = console.inputSelectedVoucherType();
+                    VoucherType voucherType = VoucherType.getSelectedVoucherType(selectedNum);
+                    if(voucherType != null){
```

Comment:
if 문 전후로는 개행을 추가하는 것이 가독성이 증가 된다고 생각해요.

---

띄어쓰기가 빠져있어서 코드 제안 드렸어요.
```suggestion
                    if (voucherType != null) {
```

Reply:
주기적인 코드 리포맷팅으로 해당 오류사항 줄이도록 하겠습니다.